.github/workflows/docs.yml

```diff
@@ -0,0 +1,68 @@
+name: Build UFL Documentation
```

Comment:
The deployment of the documentation should be in this file, and only run if a parameter deploy is true.